database/tables/workspace_logs.pg

```diff
@@ -0,0 +1,12 @@
+columns
+    date: timestamp with time zone default now()
+    id: bigint not null default nextval('workspace_logs_id_seq'::regclass)
+    log_level: enum_workspace_log_log_level
```

Comment:
I suggest changing this to having the column name be just `level` and the enum be `log_level`.